File: OOPSeminar5/MVCExample/MVCStudentApp/src/View/ViewClass.java
package View;

import java.util.List;
import java.util.Scanner;

import Controller.Interfaces.iGetView;
import Model.Core.Student;

public class ViewClass implements iGetView {

    public void printAllStudents(List<Student> students)
    {
        System.out.println("----------------Список студентов---------------------");
        for(Student s : students)
        {
            System.out.println(s);
        }
        System.out.println("-----------------------------------------------------");
    }

    @Override
    public String prompt(String msg) {
        Scanner in = new Scanner(System.in);
        System.out.println(msg);
        return in.nextLine();
    }
}

File: OOPSeminar5/python_mvc_app/view/view_class.py
from typing import List
from controller.interfaces.i_get_view import iGetView

from model.core.student import Student

class ViewClass(iGetView):
    def printAllStudent(self, students: List[Student]) -> None:
        print("----------- Список студентов -----------")
        for s in students:
            print(s)
        print("----------------------------------------")


    def prompt(self, msg: str) -> str:
        print(msg)
        return input()